src/libImaging/JpegEncode.c

```diff
@@ -137,6 +137,30 @@ ImagingJpegEncode(Imaging im, ImagingCodecState state, UINT8 *buf, int bytes) {
             /* Compressor configuration */
             jpeg_set_defaults(&context->cinfo);
 
+            /* Prevent RGB -> YCbCr conversion */
+            if (context->keep_rgb) {
+                switch (context->cinfo.in_color_space) {
+                    case JCS_RGB:
+#ifdef JCS_EXTENSIONS
+                    case JCS_EXT_RGBX:
+#endif
+                        switch (context->subsampling) {
+                            case -1:  /* Default */
+                            case 0:   /* No subsampling */
+                                break;
+                            default:
+                                /* Would subsample the GB channels, which
```

Comment:
```suggestion
                                /* Would subsample the RGB channels, which
```
?

Reply:
No, the GB channels.  We want to avoid sampling red at a higher rate than green and blue, as one might do with YCbCr.

Reply:
Maybe change it to "the G and B channels" so it looks less like a typo.

Reply:
```suggestion
                                /* Would subsample the green and blue channels, which
```

Reply:
Updated!